doc/api/fs.md

```diff
@@ -3231,11 +3231,11 @@ in that, if the `FileHandle` is not explicitly closed using the
 and will emit a process warning, thereby helping to prevent memory leaks.
 
 Instances of the `FileHandle` object are created internally by the
-`fs.promises.open()` method.
+`fsPromises.open()` method.
```

Comment:
I'd prefer it if it was referred to as simply `fs` in the examples (where it's clear it's the promise version) - that's because that's how people typically use it. Not blocking on it.

Reply:
I also prefer to use just `fs`. Let's wait and see if someone disagrees.

Reply:
I prefer `fs.promises.something` because that is clearer...if we use `fs.something` then it looks confusing for new comers if they don't read through the document and notice that that need to get the promisified version that way.

Reply:
EDIT: `s/fs.promise/fsPromises`

Reply:
After checking how the documentation is, I changed my mind and also prefer to use a different name.
Ideally, I think it should be possible to switch from the callback documentation to the Promise one instead of having one giant page with both at the same time.